### DISCUSSION TITLE
Budibase Business License and unlimited users

### DISCUSSION THREAD

**Discussion by PGallais** [Budibase/budibase#10538] [Help] [answered]
Hi,
I am new to Budibase and I am conviced by the test I have done so far. 
My need is to use Airtable as a data source to design some dashboards with this data. I am fine with that. I also see that I can define users and give them read only access to an application, access with a password protection specific to each user.
But I am not sure to understand the Premium or Business licence per user versus the "Unlimited users" mention in plans description.
If I take one License, will I be able to invite read only users to applications without paying extra licence, or do I have to pay an extra licence every time I invite a new user with read only access on an application.
Thank you

**Comment by ConorWebb96** [comment]
Hey @PGallais,

From my understanding you are able to have unlimited users. (This means there isn't a cap on how many users can be added on any of the tiers). However, a flat rate is added for each additional user added. For more information on the pricing have a look here: https://budibase.com/blog/updates/pricing-update-april-2023/ this should provide everything you need.

Hopefully this makes sense, If you have any additional questions let us know!

**Reply by PGallais** [reply]
Hy,
Thank you for your answer, but I still do not understand what happens when I add a user for read only access to an app. Will I have to subscribe a license for this read only user?
My confusion comes from the fact that the license price is per user together with the mention "unlimeted users". Does it mean that there are two types of users: "Power users" who need a licence and "Invited read only users" who do not need a license?

Thank you

**Reply by ConorWebb96** [reply, marked as answer]
Hey @PGallais,

Apologises for not explaining this better in my previous response.

Yes any new users will add to the cost paid, it doesn't matter if they have read only access or are power users - anyone who needs to log into Budibase (either as a builder, a power user or a read-only user) will need a license.

You can create public pages/screens within apps (that sit on public URLs) and anyone who accesses these won't need a license.

**Reply by PGallais** [reply]
Thanks for your answer, that's very clear
regards
Pascal
